extensions/amp-access/0.1/amp-access-client.js

```diff
@@ -126,8 +132,13 @@ export class AccessClientAdapter {
   }
 
   /** @override */
+  isPingbackEnabled() {
+    return this.isPingbackEnabled_;
+  }
+
+  /** @override */
   pingback() {
-    const promise = this.context_.buildUrl(this.pingbackUrl_,
+    const promise = this.context_.buildUrl(dev().assert(this.pingbackUrl_),
```

Comment:
Just making sure there are no cases or tests where `pingback=null` is mistakenly allowed. The way our DCE works is that `dev().assert(ARG, ...)` is entirely removed, but `ARG` expression is kept. Is this expression is not use, it will be further DCE'd on later stages.